NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -207,7 +209,16 @@ def run(
         passband: list of float, optional
             Lower and upper bound of the frequency range in which noise shall be
             added. The default (no passband specified) is [10, 1000] MHz
+        excluded_channel_ids: list, default=None
+            A list containing the channels IDs to exclude per station.
+            If None, all channels of the selected station in the detector are used.
         """
+        if excluded_channel_ids is None:
+            selected_channel_ids = station.get_channel_ids()
+            logger.info(f"Using all channels: {selected_channel_ids}")
```

Comment:
Can you make them debug messages. This would produce a lot of "noise" in simulations.